compileopts/target.go

```diff
@@ -229,7 +229,13 @@ func LoadTarget(target string) (*TargetSpec, error) {
 	}
 }
 
+// WindowsBuildNotSupportedErr is being thrown, when goos is windows and no target has been specified.
+var WindowsBuildNotSupportedErr = errors.New("Building windows binaries is currently not supported. Try specifying a target")
```

Comment:
How about this error message:

`"Building Windows binaries is currently not supported. Try specifying a different target"`

Reply:
Sounds good, i'll Update it

Reply:
If you can please squash the commit with that new change when you make together with the existing commit, that would be most appreciated.

Reply:
fixed and squashed them